robot/basestation/static/js/elements/tools/utils.js

```diff
@@ -16,3 +16,32 @@ function toggleText (button) {
     window.localStorage.setItem(button, 'text-swap')
   }
 }
+
+// Applies modulo and if the value is negative, cycle it.
+function negativeModulo (number, modulo) {
+  while (number < 0) {
+    number += modulo
+  }
+
+  number %= modulo
+  return number;
+}
+
+// Rotate element to given angle
+function rotateElement (element, angle) {
+  element.css({
+    transform: 'rotate(' + angle + 'deg)',
+    '-ms-transform': 'rotate(' + angle + 'deg)',
+    '-moz-transform': 'rotate(' + angle + 'deg)',
+    '-webkit-transform': 'rotate(' + angle + 'deg)',
+    '-o-transform': 'rotate(' + angle + 'deg)'
+  })
+}
+
+// Generates unique names for html elements with a common class
+function addIdentifiers(cssClass) {
```

Comment:
Is there any point in having this function? We can already use parent elements to identify elements with a common class. Will this come in use in the future?

Reply:
Good point. I will delete it